dcos-log/api/v2/routes.go

```diff
@@ -11,21 +11,38 @@ import (
 
 // InitRoutes inits the v1 logging routes
 func InitRoutes(v2 *mux.Router, cfg *config.Config, client *http.Client, nodeInfo nodeutil.NodeInfo) {
-	taskLogHandler := http.HandlerFunc(filesAPIHandler)
-	wrappedTaskLogHandler := middleware.Wrapped(taskLogHandler, cfg, client, nodeInfo)
+	// browse sandbox files
+	wrappedBrowseFiles := middleware.Wrapped(http.HandlerFunc(browseFiles), cfg, client, nodeInfo)
+	v2.Path("/task/frameworks/{frameworkID}/executors/{executorID}/runs/{containerID}/files/browse").Handler(wrappedBrowseFiles).Methods("GET")
```

Comment:
Here and elsewhere, I see path vars expressed as string literals in the URI path, as well as in the handlers when they need to extract values from the URI path.  does it make sense to constantize some of these?  Maybe not, but the thought came to me so i thought i'd just express it here.